packages/bits/demo/src/components/demo/checkbox-group/checkbox-group-visual-test/checkbox-group-visual-test.component.html

```diff
@@ -73,4 +73,18 @@ <h3>Disabled checkboxes within the form</h3>
             </form>
         </div>
     </div>
+    <div class="d-flex flex-column mt-5">
+        <h3>Disabled checkboxes within the disabled form</h3>
+        <form [formGroup]="testForm">
+            <nui-checkbox-group id="nui-demo-checkbox-group-in-form"
+                            name="vegetable"
+                            formControlName="checkboxGroup">
```

Comment:
it might be helpful for readability and maintenance if this group were named `disabledCheckboxGroup`. Just a suggestion.

Then, the other one (`checkboxGroup2`) could be called `enabledCheckboxGroup`.

Reply:
Sure, thank you, Blake.